docs/public/ud_format.md

```diff
@@ -113,6 +113,9 @@ the resulting string for our example would be:
 > **NB**: The complete mapping of features from Mystem to the UD format can be found in
 > the [`ud_mapping.md`](ud_mapping.md).
 
+> **HINT**: It's mandatory to use `Loc` mapping for pymystem3 `пр` case because
+> in Slavic languages this is the only case that is used exclusively in combination with prepositions.
```

Comment:
you said there is an explanation in the official docs, it would be more persuasive to put it here

Reply:
fixed